### PR TITLE
Fix deduplication in Google workspace module

### PR DESCRIPTION
**Pull request by sopmtech**
## What does this PR do?

It fixes document id field.

## Why is it important?

Avoid log duplicates

## Checklist

- [x] My code follows the style guidelines of this project
~~I have commented my code, particularly in hard-to-understand areas~~
~~I have made corresponding changes to the documentation~~
~~I have made corresponding change to the default configuration files~~
~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

**Comment by efd6** [comment]
@sopmtech Please sign the CLA so that we can take a look at this.

**Comment by sopmtech** [comment]
@efd6 I have already signed CLA. Unfortunately, right after the pull request.

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -53,6 +53,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Fix type mapping of client.as.number in okta module. {pull}31676[31676]
 - Fix last write pagination commit checkpoint on `aws-s3` input for s3 direct polling when using the same bucket and different list prefixes. {pull}31776[31776]
 - If a file is ignored by `filestream` because of ignore_older settings, when it is updated, only the new lines are shipped to the output. {issue}31924[31924] {pull}31972[31972]
+- Fix deduplication in Google workspace module. {pull}31898[31898]
```
[the remark this answers is not in the document]
Comment (sopmtech) [reply CHANGELOG.next.asciidoc]: Check out a new version of the changelog entry.

**Comment by kcreddy** [comment]
@Mergifyio backport 7.17

**Comment by joeafari** [comment]
Hey @kcreddy 
Do you know if the automatic backport to 7.17 was done? Asking because I don't see any reference to it in this issue.

**Pull request merged**